<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1000</width>
                <height>700</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Ëà™Áè≠Á•®Âä°ÁÆ°ÁêÜÁ≥ªÁªü - FlyOver</string>
        </property>
        <widget class="QWidget" name="centralWidget">
            <layout class="QHBoxLayout" name="mainLayout">
                <item>
                    <widget class="QFrame" name="sideBar">
                        <property name="frameShape">
                            <enum>QFrame::Shape::StyledPanel</enum>
                        </property>
                        <layout class="QVBoxLayout" name="sideLayout">
                            <item>
                                <widget class="QLabel" name="logoLabel">
                                    <property name="styleSheet">
                                        <string notr="true">font-size: 20px; font-weight: bold; color: #0078d7; margin: 20px 0;</string>
                                    </property>
                                    <property name="text">
                                        <string>‚úàÔ∏è FlyOver</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignmentFlag::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btnHome">
                                    <property name="styleSheet">
                                        <string notr="true">text-align: left; padding: 12px 20px;</string>
                                    </property>
                                    <property name="text">
                                        <string>üè†‰∏ª¬† ¬† ¬† ¬†È°µ</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btnFlightQuery">
                                    <property name="styleSheet">
                                        <string notr="true">text-align: left; padding: 12px 20px;</string>
                                    </property>
                                    <property name="text">
                                        <string>‚úàÔ∏èËà™Áè≠Êü•ËØ¢</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btnTicketBooking">
                                    <property name="styleSheet">
                                        <string notr="true">text-align: left; padding: 12px 20px;</string>
                                    </property>
                                    <property name="text">
                                        <string>üõ´Ë¥≠Á•®Á≥ªÁªü</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="travelbutton">
                                    <property name="styleSheet">
                                        <string notr="true">text-align: left; padding: 12px 20px;</string>
                                    </property>
                                    <property name="text">
                                        <string>üåçÊóÖË°åÂä®ÊÄÅ</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btnOrderManagement">
                                    <property name="styleSheet">
                                        <string notr="true">text-align: left; padding: 12px 20px;</string>
                                    </property>
                                    <property name="text">
                                        <string>üßæËÆ¢ÂçïÁÆ°ÁêÜ</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btnDataAnalytics">
                                    <property name="styleSheet">
                                        <string notr="true">text-align: left; padding: 12px 20px;</string>
                                    </property>
                                    <property name="text">
                                        <string>üìäÊï∞ÊçÆÂàÜÊûê</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btnProfile">
                                    <property name="styleSheet">
                                        <string notr="true">text-align: left; padding: 12px 20px;</string>
                                    </property>
                                    <property name="text">
                                        <string>üë§ÊàëÁöÑ‰ø°ÊÅØ</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btnAIService">
                                    <property name="styleSheet">
                                        <string notr="true">text-align: left; padding: 12px 20px;</string>
                                    </property>
                                    <property name="text">
                                        <string>üí¨¬† AI ÂÆ¢Êúç</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                </item>
                <item>
                    <widget class="QStackedWidget" name="stackedWidget">
                        <property name="styleSheet">
                            <string notr="true">background-color: #1e1e1e;</string>
                        </property>
                        <widget class="QWidget" name="pageHome">
                            <layout class="QVBoxLayout" name="homeLayout">
                                <item>
                                    <widget class="QLabel" name="labelWelcome">
                                        <property name="styleSheet">
                                            <string notr="true">font-size: 22px; font-weight: bold; margin-top: 80px;</string>
                                        </property>
                                        <property name="text">
                                            <string>Ê¨¢Ëøé‰ΩøÁî®Ëà™Áè≠Á•®Âä°ÁÆ°ÁêÜÁ≥ªÁªü - FlyOver</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignmentFlag::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </widget>
                        <widget class="QWidget" name="pageFlights">
                            <layout class="QVBoxLayout" name="flightsLayout"/>
                        </widget>
                        <widget class="QWidget" name="pageTravel">
                            <layout class="QVBoxLayout" name="travelLayout">
                                <item>
                                    <widget class="QLabel" name="labelTravel">
                                        <property name="text">
                                            <string>ÊóÖË°åÂä®ÊÄÅÈ°µÈù¢</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignmentFlag::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </widget>
                        <widget class="QWidget" name="pageOrders">
                            <layout class="QVBoxLayout" name="ordersLayout">
                                <item>
                                    <widget class="QLabel" name="labelOrders">
                                        <property name="text">
                                            <string>ËÆ¢ÂçïÁÆ°ÁêÜÈ°µÈù¢</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignmentFlag::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </widget>
                        <widget class="QWidget" name="pageProfile">
                            <layout class="QVBoxLayout" name="profileLayout">
                                <item>
                                    <widget class="QLabel" name="labelProfile">
                                        <property name="text">
                                            <string>‰∏™‰∫∫‰ø°ÊÅØÈ°µÈù¢</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignmentFlag::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </widget>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menuBar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>1000</width>
                    <height>21</height>
                </rect>
            </property>
            <widget class="QMenu" name="menuFlights">
                <property name="title">
                    <string>Ëà™Áè≠ÁÆ°ÁêÜ(&amp;F)</string>
                </property>
                <addaction name="actionFlightManager"/>
            </widget>
            <widget class="QMenu" name="menuOrders">
                <property name="title">
                    <string>ËÆ¢ÂçïÁÆ°ÁêÜ(&amp;O)</string>
                </property>
                <addaction name="actionViewOrders"/>
            </widget>
            <widget class="QMenu" name="menuUser">
                <property name="title">
                    <string>Áî®Êà∑‰∏≠ÂøÉ(&amp;U)</string>
                </property>
                <addaction name="actionProfile"/>
                <addaction name="actionLogout"/>
            </widget>
            <widget class="QMenu" name="menuHelp">
                <property name="title">
                    <string>Â∏ÆÂä©(&amp;H)</string>
                </property>
                <addaction name="actionAbout"/>
            </widget>
            <widget class="QMenu" name="menuSettings">
                <property name="title">
                    <string>ËÆæÁΩÆ(&amp;S)</string>
                </property>
                <addaction name="actionToggleTheme"/>
            </widget>
            <addaction name="menuFlights"/>
            <addaction name="menuOrders"/>
            <addaction name="menuUser"/>
            <addaction name="menuSettings"/>
            <addaction name="menuHelp"/>
        </widget>
        <widget class="QToolBar" name="mainToolBar">
            <attribute name="toolBarArea">
                <enum>TopToolBarArea</enum>
            </attribute>
            <attribute name="toolBarBreak">
                <bool>false</bool>
            </attribute>
        </widget>
        <widget class="QStatusBar" name="statusBar"/>
        <action name="actionFlightManager">
            <property name="text">
                <string>Ëà™Áè≠ÁÆ°ÁêÜ</string>
            </property>
        </action>
        <action name="actionManageFlights">
            <property name="text">
                <string>Ëà™Áè≠ÂàóË°®</string>
            </property>
        </action>
        <action name="actionViewOrders">
            <property name="text">
                <string>Êü•ÁúãËÆ¢Âçï</string>
            </property>
        </action>
        <action name="actionProfile">
            <property name="text">
                <string>‰∏™‰∫∫‰ø°ÊÅØ</string>
            </property>
        </action>
        <action name="actionLogout">
            <property name="text">
                <string>ÈÄÄÂá∫ÁôªÂΩï</string>
            </property>
        </action>
        <action name="actionAbout">
            <property name="text">
                <string>ÂÖ≥‰∫éÁ≥ªÁªü</string>
            </property>
        </action>
        <action name="actionToggleTheme">
            <property name="text">
                <string>ÂàáÊç¢Ê∑±Ëâ≤/ÊµÖËâ≤‰∏ªÈ¢ò</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+T</string>
            </property>
        </action>
    </widget>
    <layoutdefault spacing="6" margin="11"/>
    <resources/>
    <connections/>
</ui>
